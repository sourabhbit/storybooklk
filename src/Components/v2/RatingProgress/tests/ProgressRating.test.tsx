import { render, screen } from '@testing-library/react'
import ProgressRatings from '../index'

const props = {
	rating: 4.5,
	max: 5,
}

describe('<ProgressRatings />', () => {
	let wrapper: any = null

	beforeEach(() => {
		const { asFragment } = render(<ProgressRatings {...props} />)
		wrapper = asFragment
	})

	afterAll(() => {
		wrapper = null
	})

	it('Should take a snapshot', () => {
		expect(wrapper()).toMatchSnapshot()
	})
	it('Should render Progress Rating component', () => {
		const container = screen.getByTestId('progress-test')
		expect(container).toBeInTheDocument()
	})
})
