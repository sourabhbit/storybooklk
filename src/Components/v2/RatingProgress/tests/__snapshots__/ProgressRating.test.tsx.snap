// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressRatings /> Should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL"
    data-testid="progress-test"
  >
    <div
      class="sc-gsDKAQ dZaqmM"
    >
      <progress
        max="5"
        value="4.5"
      >
        4.5
      </progress>
    </div>
  </div>
</DocumentFragment>
`;
